test/uk/gov/hmrc/ctcguaranteebalancerouter/services/BalanceRetrievalServiceSpec.scala

```diff
@@ -114,18 +96,18 @@ class BalanceRetrievalServiceSpec extends AnyFreeSpec with Matchers with ScalaCh
         }
     }
 
-    "on a not found, return a Left of BalanceRetrievalError.Unexpected" in forAll(
+    "on a not found, return a Left of BalanceRetrievalError.BalanceRetrievalServiceSpec.scala" in forAll(
```

Comment:
Remove the `.scala`!

Reply:
Good spot! Should've been BalanceRetrievalError.GrnNotFound haha

Reply:
Yes, I... _totally_ spotted that it was more than just that, I was clearly testing you... yes...